pkg/kademlia/routing_helpers_test.go

```diff
@@ -349,24 +308,25 @@ func TestNodeIsWithinNearestK(t *testing.T) {
 			closest: true,
 		},
 		{testID: "C",
-			nodeID:  storj.NodeID{255, 255},
+			nodeID:  storj.NodeID{255, 255}, //closest to self
 			closest: true,
 		},
 		{testID: "D",
 			nodeID:  storj.NodeID{191, 255},
-			closest: true,
+			closest: false,
 		},
 		{testID: "E",
 			nodeID:  storj.NodeID{133, 255},
 			closest: false,
 		},
 	}
-
+	xors := []storj.NodeID{}
```

Comment:
are the `xors` used somehow?

Reply:
good catch, that's residual. will remove